README.md

```diff
@@ -2,8 +2,9 @@
 
 A multi-system emulator written in C#. As well as quality-of-life features for casual players, it also has recording/playback and debugging tools, making it the first choice for TASers (Tool-Assisted Speedrunners).
 
-[![GitHub latest release](https://img.shields.io/github/release/TASVideos/BizHawk.svg?logo=github&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/releases/latest)
-[![dev builds | AppVeyor](https://img.shields.io/badge/dev_builds-AppVeyor-orange.svg?logo=appveyor&logoColor=333333&style=popout)](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/history)
+[![(latest) release | GitHub](https://img.shields.io/github/release/TASVideos/BizHawk.svg?logo=github&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/releases/latest)
+[![latest dev build | AppVeyor](https://img.shields.io/badge/latest_dev_build-AppVeyor-orange.svg?logo=appveyor&logoColor=333333&style=popout)](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts)
+[![latest dev build | GitLab CI](https://img.shields.io/badge/latest_dev_build-GitLab_CI-orange.svg?logo=gitlab&logoColor=333333&style=popout)](https://gitlab.com/TASVideos/BizHawk/pipelines/master/latest)
```

Comment:
I'd love both appveyor and gitlab links to point to latest master artifact, but if not, they at the very least should be consistent, and point to all commits. Here they're different: appveyor links all commits, and gitlab links latest master.

Reply:
The list to all branches is on each, and the link to latest is only for GitLab and is clearly labelled as `master`.

Reply:
Why not link appveyor latest too? It was linked in the Testing section before, now there's no such link at all. And it's the most sensible link for people not looking for a specific commit. When we need people to test "nightlies" often, it should be easily accessible.

Reply:
There's no redirect link on AppVeyor to jump to `master`, so it would be the most recent from any branch. That's what line 6 in the current readme does.

Reply:
Current readme:

> ### Testing
> 
> * [Latest development build](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts)

Since appveyor only builds master, that's the link one needs to get the latest dev build without any searching.

Reply:
I forgot we changed AppVeyor to only build master...